VCFv4.3.tex

```diff
@@ -1413,10 +1413,25 @@ \subsection{Overall file organization}
 
 \subsection{Header}
 
-The BCF2 header begins with the ``BCF2 magic'' 5 bytes that encode {\tt BCF\em XY} where {\em X} and {\em Y} are bytes indicating the major number (currently 2) and the minor number (currently 2).
-This magic can be used to quickly examine the file to determine that it's a BCF2 file.
-Immediately following the BCF2 magic is an unsigned 4-byte integer containing the length of the header text which includes the NUL terminator character.
-Follow the standard NUL-terminated VCF header lines in text format, beginning with \verb|##fileformat=VCFvX.Y|.
+The BCF2 header contains the following items:
+
+{\small
+\begin{tabular}{|l | l | p{30em} | } \hline
+\textbf{Field} & \textbf{Type} & \textbf{Notes} \\ \hline
+magic          & char[3]     & The characters 'B', 'C', 'F' \\ \hline
```

Comment:
```suggestion
magic          & char[3]     & The characters `B', `C', `F' \\ \hline
```

And perhaps put the literal characters in `\tt` too.

Reply:
Hmmm.  What would be better?
```
The characters `{\tt B}', `{\tt C}', `{\tt F}'
```
or
```
The characters ``{\tt BCF}''
```

Reply:
It's `char[3]`. It's pretty clear "BCF" means those 3 chars in that order without being tediously explicit.